lib/floe.rb

```diff
@@ -44,6 +44,17 @@ class Error < StandardError; end
   class InvalidWorkflowError < Error; end
   class InvalidExecutionInput < Error; end
 
+  class ExecutionError < Error
+    attr_reader :floe_error
+
+    def initialize(message, floe_error = "States.RuntimeError")
```

Comment:
As Adam asked elsewhere, is States.RuntimeError an actual error?

Reply:
https://github.com/ManageIQ/floe/pull/189#pullrequestreview-2177893255 it isn't listed in the states-language spec but it is in the AWS StepFunctions docs and he fixed it to be `States.Runtime` to match the docs there